django-backend/fecfiler/settings/base.py

```diff
@@ -380,3 +380,5 @@ def get_logging_processors():
 )
 
 TEST_RUNNER = "fecfiler.test_runner.CustomTestRunner"
+
+ENABLE_DEVELOPER_COMMANDS = env.get_credential("ENABLE_DEVELOPER_COMMANDS")
```

Comment:
Can you make the right side evaluate to a boolean?  Right now it's pulling a string which is truthy unless "" or None

Reply:
Oh good catch! Yep got that fixed now.